Source/Scene/BillboardCollection.js

```diff
@@ -255,6 +266,16 @@ define([
                 return that._textureAtlas.texture;
             }
         };
+
+        this._removeEventFunc = undefined;
+        if (defined(this._globe)) {
+            this._removeEventFunc = this._globe._surface.tileRenderedEvent.addEventListener(function(tile) {
+                var tileList = that._renderedTileList;
+                if (tileList.indexOf(tile) === -1) {
```

Comment:
How often does this linear search happen?  And in what case?  When a tile went from visible to invisible to visible with it being processed on its first visibility event?